Automattic/woocommerce-payments
# Add payment method logos to card label

Fixes #9826 
Closes #9660

#### Changes proposed in this Pull Request

This adds a payment method logos component to the card label on the checkout block. There's another component to this that involves Stripe removing the payment method logo animations from the card number field. For now, this has been implemented on a dev account that we can use for testing. Before this can be merged, we'll need a couple things:

1. Completion of #9660. This reduces the card label text to "Cards" which this PR depends on. 
2. A rollout plan coordinated with Stripe.

Because payment method animations will be disabled for all WooPayments account, we'll need to ensure we're coordinated with Stripe on the removal. For major release, I think the earliest date would be Feb 5, 2025. However, we could always do a patch release if tighter alignment with Stripe is required.









#### Testing instructions

Gated Dev account - p1732130572346229/1732125470.634879-slack-C043JN7AJHY

Note: if #9660 hasn't yet been merged, please update the card label text in the browser to read "Cards". This is important because this PR depends on that shorter label to display more card brand logos, especially on mobile.

1. Migrate your account to the gated dev account
3. Add a product to the cart and visit the checkout block
4. Notice the card icons right aligned on the Credit card / debit card label
5. Repeat the above 2 steps on classic checkout





### Blocks Checkout
![image](https://github.com/user-attachments/assets/06b38702-31ca-4b16-a68a-16784fc1d04d)

### Classic Checkout
![image](https://github.com/user-attachments/assets/3c85b62c-503c-4b9b-bee4-ca1b3a34e811)

-------------------


**Post merge**
